applications/GeoMechanicsApplication/custom_retention/retention_law_factory.h

```diff
@@ -53,6 +53,8 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) RetentionLawFactory
             if (RetentionLawName == "SaturatedBelowPhreaticLevelLaw")
                 return make_unique<SaturatedBelowPhreaticLevelLaw>();
 
+            if (RetentionLawName == "PressureFilterLaw") return make_unique<SaturatedLaw>();
```

Comment:
I think like this we will use Kratos::make_unique. I'd prefer to use std::make_unique
```suggestion
            if (RetentionLawName == "PressureFilterLaw") return std::make_unique<SaturatedLaw>();
```